flink-runtime/src/main/java/org/apache/flink/runtime/heartbeat/HeartbeatServices.java

```diff
@@ -40,7 +40,7 @@
 
 	public HeartbeatServices(long heartbeatInterval, long heartbeatTimeout) {
 		Preconditions.checkArgument(0L < heartbeatInterval, "The heartbeat interval must be larger than 0.");
-		Preconditions.checkArgument(heartbeatInterval <= heartbeatTimeout, "The heartbeat timeout should be larger or equal than the heartbeat timeout.");
+		Preconditions.checkArgument(heartbeatInterval <= heartbeatTimeout, "The heartbeat interval should be larger or equal than the heartbeat timeout.");
```

Comment:
From the actual comparison, it looks like the second `timeout` should be changed to `interval` at the end of message.

Reply:
`The heartbeat interval should be larger or equal than the heartbeat timeout` -> `The heartbeat timeout should be larger or equal than the heartbeat interval`